ts/kpt-functions/src/types.ts

```diff
@@ -166,6 +166,36 @@ export class Configs {
     return this.functionConfig;
   }
 
+  /**
+   * Returns the map of data values if functionConfig is of kind ConfigMap.
+   *
+   * Throws a FunctionConfigError if functionConfig is undefined OR
+   * if the kind is not a v1/ConfigMap.
+   */
+  getFunctionConfigMap() {
+    const cm = this.getFunctionConfig();
+    if (cm === undefined) {
+      throw new FunctionConfigError(
+        `functionConfig expected, instead undefined`
+      );
+    }
+    if (!isConfigMap(cm)) {
+      throw new FunctionConfigError(
+        `functionConfig expected to be of kind ConfigMap, instead got: ${cm.kind}`
+      );
+    }
+    const configMap = new Map<string, string>();
+    if (!cm.data) {
```

Comment:
wouldn't this work without this conditional? The for loop never iterates

Reply:
Removed conditional